ansible/playbooks/fully-patch-system.yaml

```
@@ -20,3 +20,5 @@
     - name: Reboot after patch
       ansible.builtin.reboot:
         reboot_timeout: "{{ use_reboottimeout | int }}"
```

Maybe if you like just change the order

```suggestion
        msg: "Reboot initiated by Ansible - after full patch system"
        reboot_timeout: "{{ use_reboottimeout | int }}"
        connect_timeout: 5
```

---

Changed the order according to your comments. It looks better :) , thanks.